fastlane/screenshots.json

```diff
@@ -0,0 +1,63 @@
+{
+	"version": 0.1,
+	"shadow_offset": 0,
+	"stylesheet": "appstoreres/assets/styles/style.css",
+	"devices": [
+		{
+			"name": "iPhone 8 Plus",
+			"canvas_size": [1242,2208],
+			"text_size": [994, 310],
+			"text_offset": [120, 120],
+			"font_size": "105px",
+			"screenshot_size": [826, 1468],
+			"screenshot_offset": [208, 740],
+			"device_frame": "appstoreres/assets/iphone_8_plus.png",
+			"device_frame_size": [1242, 2208],
+			"device_frame_offset": [0, 0]
+		}
+	],
+	"entries": [
+		{
+			"device": "iPhone 8 Plus",
+			"filename": "iPhone 8 Plus-01.png",
+			"background": "#FFFFFF",
+			"screenshot": "iPhone 8 Plus-1-note.png",
+			"text": "metadata/{locale}/app_store_screenshot_1.txt"
+		},
+		{
+			"device": "iPhone 8 Plus",
+			"filename": "iPhone 8 Plus-02.png",
+			"background": "#FFFFFF",
+			"screenshot": "iPhone 8 Plus-2-all-notes.png",
+			"text": "metadata/{locale}/app_store_screenshot_2.txt"
+		},
+		{
+			"device": "iPhone 8 Plus",
```

Comment:
We should probably dismiss the contacts popup for this screenshot :) 
![iPhone 8 Plus-03](https://user-images.githubusercontent.com/1123407/78153746-a543f780-73f8-11ea-8c8c-04d7daa0dddf.png)

Reply:
Good catch. Thanks!

Some of my tests dismissed it automatically, but other didn't. I remember reading Fastlane should do this for us, but either I'm wrong or there's something escaping it...

All this to say. I've fixed this earlier, before reading this review. I'll cherry pick 🍒 👍